content/management/configuration-api/v3.3/index.mdx

```diff
@@ -1794,6 +1799,7 @@ Registers a new private property for a given **Client Id**.
 | `range` __*__              | No                  | `object`   | Range of values that properties can be set to                                         |
 | `range.from`               | No                  | `int`      | Only values **equal** or **greater** than this parameter can be set to this property  |
 | `range.to`                 | No                  | `int`      | Only values **equal** or **lower** than this parameter can be set to this property    |
+| `default_value`            | No                  | `type`     | Default value of property, validated by **domain** or **range** if one exists         |
```

Comment:
```suggestion
| `default_value`            | No                  | `type`     | Default value of a property; validated by **domain** or **range**, if one exists.         |
```

Reply:
Maybe it's worth mentioning that it's only possible for group and license (property locations) 🤔 ?